scripts/quests/ahtUrhgan/Led_Astray.lua

```diff
@@ -0,0 +1,250 @@
+-----------------------------------
+-- Led Astry
+-- Log: 6, Quest: 65
+-- Mhasbaf:         !pos 54.701 -6.999 11.387 50
+-- Whitegate Region !pos 71 0 9.299 50
+-- Tohka Telposkha: !pos 22.1 0 22.759 50
+-- Rubahah:         !pos -96.746 0 -14.701 50
+-- Cacaroon:        !pos -72.026 0.000 -82.337 50
+-- Whitegate Region !pos 12.5 -94 2 50
+-- Tataroon:        !pos -25.189 0 -39.022 53
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/npc_util')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/titles')
+-----------------------------------
```

Comment:
Should probably include scripts/globals/zone.lua here, since we use it in the tables below